app/views/videos/index.js.erb

```diff
@@ -0,0 +1 @@
+$('#video-list').html("<%= escape_javascript(render('video_list')) %>");
```

Comment:
Same deal. At this point, I'm putting tags that you shouldn't see after the update.